11:47:49.598 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:47:50.200 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:47:51.105 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:47:51.130 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
11:52:22.870 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
11:52:23.897 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
11:52:23.935 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
11:52:24.966 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
11:52:25.024 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
11:52:26.122 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
11:52:26.123 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.LocalFileConfigRepository, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/./config_cache/search-master/config-cache/search-master+default+application.properties][0;39m 
11:52:26.195 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
11:52:27.257 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
11:52:27.258 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mInit Apollo Local Config failed - namespace: application, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/./config_cache/search-master/config-cache/search-master+default+application.properties].[0;39m 
11:52:27.259 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mCould not load config for namespace application from Apollo, please check whether the configs are released in Apollo! Return default value now![0;39m 
11:52:29.192 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:52:29.792 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:52:30.683 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:52:30.707 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
11:56:32.469 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:56:32.651 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.foundation.internals.provider.DefaultApplicationProvider[0;39m - [31mapp.id is not available from System Property and /META-INF/app.properties. It is set to null[0;39m 
11:56:32.733 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultMetaServerProvider[0;39m - [31mCould not find meta server address, because it is not available in neither (1) JVM system property 'apollo.meta', (2) OS env variable 'APOLLO_META' (3) property 'apollo.meta' from server.properties nor (4) property 'apollo.meta' from app.properties[0;39m 
11:56:32.735 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.core.MetaDomainConsts[0;39m - [31mMeta server address fallback to http://apollo.meta for env UNKNOWN, because it is not available in all MetaServerProviders[0;39m 
11:56:32.736 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.util.ConfigUtil[0;39m - [31mapp.id is not set, please make sure it is set in classpath:/META-INF/app.properties, now apollo will only load public namespace configurations![0;39m 
11:56:36.898 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:38.914 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:40.920 [[1;33mApollo-RemoteConfigLongPollService-1[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigLongPollService[0;39m - [31mLong polling failed, will retry in 1 seconds. appId: ApolloNoAppIdPlaceHolder, cluster: default, namespaces: application, long polling url: null, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:42.930 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:42.931 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.LocalFileConfigRepository, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/search-master/target/classes/config-cache/ApolloNoAppIdPlaceHolder+default+application.properties][0;39m 
11:56:44.935 [[1;33mApollo-RemoteConfigLongPollService-1[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigLongPollService[0;39m - [31mLong polling failed, will retry in 2 seconds. appId: ApolloNoAppIdPlaceHolder, cluster: default, namespaces: application, long polling url: null, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:46.942 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Get config services failed from http://apollo.meta/services/config?appId=ApolloNoAppIdPlaceHolder&ip=192.168.1.193 [Cause: Could not complete get operation [Cause: apollo.meta]][0;39m 
11:56:46.943 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mInit Apollo Local Config failed - namespace: application, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/search-master/target/classes/config-cache/ApolloNoAppIdPlaceHolder+default+application.properties].[0;39m 
11:56:46.991 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mCould not load config for namespace application from Apollo, please check whether the configs are released in Apollo! Return default value now![0;39m 
11:56:47.374 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:56:48.244 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:56:54.582 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:56:55.041 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:56:55.787 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:56:55.825 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
11:57:51.770 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:59:09.996 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:59:10.547 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:59:11.400 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:59:11.424 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
11:59:36.250 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
11:59:36.809 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
11:59:37.637 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
11:59:37.673 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
12:00:20.845 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.mybatis.spring.mapper.ClassPathMapperScanner[0;39m - [31mNo MyBatis mapper was found in '[com.master]' package. Please check your configuration.[0;39m 
12:00:21.321 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
12:00:22.096 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mo.s.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext[0;39m - [31mException encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is org.springframework.boot.autoconfigure.jdbc.DataSourceProperties$DataSourceBeanCreationException: Failed to determine a suitable driver class[0;39m 
12:00:22.116 [[1;33mmain[0;39m] [1;31mERROR[0;39m [1;32morg.springframework.boot.diagnostics.LoggingFailureAnalysisReporter[0;39m - [1;31m

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles prd are currently active).
[0;39m 
14:44:27.963 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
14:45:46.280 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:45:53.174 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:45:59.644 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:46:48.514 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:46:56.912 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:47:44.650 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
14:47:49.218 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:48:15.753 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:48:35.875 [[1;33mThreadPoolTaskExecutor-3[0;39m] [1;31mERROR[0;39m [1;32mcom.service.base.BaseService[0;39m - [1;31mNo ss result got from [SEARCH_MASTER_QU]-[http://internalapi.aigauss.com/baymaxentity/api/entities], got result as:[{}][0;39m 
14:48:35.897 [[1;33mThreadPoolTaskExecutor-6[0;39m] [1;31mERROR[0;39m [1;32mcom.service.base.BaseService[0;39m - [1;31mInvalidated param:[QueryRequest(queryConfig=QueryConfig(globalRate=1.0, rateMapping={BULLETIN_CALLBACK=1.0, SEARCH_MASTER_QU=1.0, EARNING_CALLBACK=1.0, RESEARCH_CALLBACK=1.0, SEARCH_MASTER_SEARCHER=1.0, SEARCH_MASTER=1.0, NEWS_CALLBACK=1.0}), coreQuery=WorkerCoreQuery(groupSize=10, dataSources=[], querySegments=[], stockList=[], projectList=[]))][0;39m 
14:49:21.252 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
14:49:25.403 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
14:50:06.059 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
14:50:10.135 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:02:18.000 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:02:39.668 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:49:13.083 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
15:49:39.723 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:49:44.024 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:49:49.214 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:49:51.315 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:49:52.173 [[1;33mXNIO-1 task-1[0;39m] [1;31mERROR[0;39m [1;32mcom.common.CommonExceptionHandler[0;39m - [1;31mError happened on url:[/v1/api/msSearch][0;39m 
15:50:43.818 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
15:54:31.856 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
17:14:35.032 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
17:14:36.079 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
17:14:36.189 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
17:14:37.269 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
17:14:37.368 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
17:14:38.465 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
17:14:38.466 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.AbstractConfigRepository[0;39m - [31mSync config failed, will retry. Repository class com.ctrip.framework.apollo.internals.LocalFileConfigRepository, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/./config_cache/search-master/config-cache/search-master+default+application.properties][0;39m 
17:14:38.529 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.RemoteConfigRepository[0;39m - [31mLoad config failed, will retry in 1 SECONDS. appId: search-master, cluster: default, namespaces: application[0;39m 
17:14:39.587 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.LocalFileConfigRepository[0;39m - [31mSync config from upstream repository class com.ctrip.framework.apollo.internals.RemoteConfigRepository failed, reason: Load Apollo Config failed - appId: search-master, cluster: default, namespace: application, url: http://10.0.100.251:8080/configs/search-master/default/application?ip=192.168.1.193 [Cause: [status code: 404] Could not find config for namespace - appId: search-master, cluster: default, namespace: application, please check whether the configs are released in Apollo!][0;39m 
17:14:39.588 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mInit Apollo Local Config failed - namespace: application, reason: Load config from local config failed! [Cause: Cannot read from local cache file /Users/stevenchen/working/sourcecode/mainsearch/mssearcher/./config_cache/search-master/config-cache/search-master+default+application.properties].[0;39m 
17:14:39.590 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mcom.ctrip.framework.apollo.internals.DefaultConfig[0;39m - [31mCould not load config for namespace application from Apollo, please check whether the configs are released in Apollo! Return default value now![0;39m 
17:14:41.213 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32morg.springframework.boot.context.properties.PropertySourcesDeducer[0;39m - [31mMultiple PropertySourcesPlaceholderConfigurer beans registered [propertySourcesPlaceholderConfigurer, org.springframework.context.support.PropertySourcesPlaceholderConfigurer], falling back to Environment[0;39m 
17:14:41.586 [[1;33mmain[0;39m] [31mWARN [0;39m [1;32mio.undertow.websockets.jsr[0;39m - [31mUT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used[0;39m 
